language-design.md

```diff
@@ -47,7 +85,9 @@ In what way is an API a language?
 
 **Response**
 
-
+APIs differ from DSLs in that APIs don't implement their own syntax, and instead borrow it
+from the host language, however they still have vocabulary and grammar. You still must use 
+the vocabulary of the API in the way that it allows.
```

Comment:
This is a very cool insight!

What is syntax? I agree that for some definitions of syntax (operators, keywords, etc) API's are limited by the host languages (at least all host languages that I know of). However, what about syntax as a set of rules for how language primitives can be combined? How is that restricted by the host language?